tests/harvester/test_file.py

```diff
@@ -42,6 +42,24 @@ def test_filename(alto_file, alto_filename):
     assert alto_file.filename == alto_filename
 
 
+def test_erroneus_filename(alto_file_with_erroneous_name):
+    """
+    Ensure that an erroneous ALTO filename raises an error.
+    """
+    with pytest.raises(AttributeError, match=r".* alto.xml .*") as e:
```

Comment:
If we don't make any assertions about the raised error, we don't need the `as e` part, now it's just an unused variable.

Reply:
This was leftover from testing which exception potential errors raise. I'll remove it to be extra neat.